cmd/kaspawallet/keys/keys.go

```diff
@@ -367,7 +367,7 @@ func (d *File) detectNumThreads(password []byte, encryptedMnemonic *EncryptedMne
 
 		_, err := decryptMnemonic(numThreadsGuess, encryptedMnemonic, password)
 		if err != nil {
-			const maxTries = 32
+			const maxTries = 256
```

Comment:
The max number of threads that argon2.IDKey can get is 255

Reply:
```suggestion
			const maxTries = 255
```